.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor
 jsonnetfile.lock.json
+.idea
```

Comment:
IMO this should go in your ~/.gitignore rather than in every repo.

Reply:
Well I assume it'd be helpful for everyone using Jetbrains' products but yeah I can revert that change.